docs/docs/faq.md

```diff
@@ -76,6 +76,10 @@ Therefore in ZMK, [board](#what-is-a-board) and [shield](#what-is-a-shield) are
 
 Please note, many keyboards only have a single PCB which includes the “brains” (MCU) onboard. In ZMK, these have no [shield](#what-is-a-shield), only a [board](#what-is-a-board).
 
+### Does ZMK Support Wired Split?
```

Comment:
```suggestion
### Does ZMK support wired split?
```
Just noticed this is necessary to fit in with the style of the FAQ page.  Would you mind squashing it in please?

Reply:
Thanks!